doc/design/cluster_train/pserver_client.md

```diff
@@ -0,0 +1,157 @@
+# Design Doc: The Client Library of Parameter Server
+
+For an overview of trainer's role, please refer to [distributed training design doc](README.md). In this design doc, we will discuss the parameter server's client library, which will manage communication with parameter servers. The library will be implemented in [Go](https://golang.org/) and made available as a static or dynamic library with a C header file.
+
+## Parameter Partition
+
+Each parameter will be partitioned into parameter chunks to make the parameters evenly distributed on parameter servers. The partition is done automatically by the client library. The *sparse parameter* require a little different treatment:
```

Comment:
Hmm, we should unify the Parameter Partition name with ParameterServer, I just wrote `ParameterBlock` as the Paddle already have.